.github/workflows/events-nightly.yml

```
@@ -123,7 +125,7 @@ jobs:
       matrix:
         # the config used in '.github/workflows/ci_test-conda.yml'
         python_version: ["3.8"]
-        pytorch_version: ["1.6", "1.7", "1.8", "1.9", "1.10"]
+        pytorch_version: ["1.7.1", "1.8", "1.9", "1.10"]
```

```suggestion
        pytorch_version: ["1.7", "1.8", "1.9", "1.10"]
```

That should work since we test with 3.8 here, no?